optking/hessian.py

```diff
@@ -89,15 +89,18 @@ def from_file(filename):
         if '.json' == filename[:-5]:
             result = json.load(f)
             hess = result['return_result']
+            natom = len(result['molecule']['symbols'])
         else:
             # 2D split. Cast everything to floats and convert back to 2D list from map.
             lines = f.readlines()
-            hess = [list(map(float, line.split())) for line in lines]
-
+            hess = [list(map(float, line.split())) for line in lines[1:]]
+            natom = int(lines[0].split()[1])
```

Comment:
Is there some standard that this is number of atoms?  I don't mind, but wondering.  The old file15/hessian used to have something weird like 3*N or 6*N here.